### PR TITLE
Issue/130 tardis testing

### PR DESCRIPTION
**Pull request by zzgvh**
I'd like comments about the coding itself, as well as ideas on how to test the Tardis even more.

### CODE REVIEW

File: backend/test/org/akvo/dash/tardis_test.clj [hunk 1]
```diff
@@ -34,39 +35,36 @@
...
+        (let [test-delete (fn [db_uri]
+                            (delete-dataset-by-id db_uri dataset)
+                            (is (=
+                                  0
+                                  ((count-history-dataset-by-ts db_uri {:now (sql-now)}) :count)))
+                            (is (=
+                                  2
+                                  ((count-all-history-datasets db_uri) :count))))]
+          (for-all-tenants test-delete)))
       )))
```
Comment (kardan) [inline backend/test/org/akvo/dash/tardis_test.clj]: These trailing parens can be moved prior line.

**Comment by kardan** [comment]
Minor style opinions which was covered in a call. But otherwise works fine when executed via
```bash
lein test :all
```
But there is an issue when testing via a REPL & editor and the system-fixture which always thinks it's running non test mode and try to use the normal db, not the test ones the provisioning script created. 

To be followed up.

**Comment by kardan** [comment]
@zzgvh ping, you worked more on this?

**Comment by iperdomo** [comment]
@kardan i think we should close the PR ? We're not actively working on this. Not sure if the code is in _good state_ to be merged?

**Pull request closed without merging**